com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -691,36 +660,65 @@ public int GetVideoMemoryCost()
 
         void RemoveCell(Cell cell)
         {
-            if (cell.loaded)
+            if (cells.TryGetValue(cell.index, out var cellInfo))
             {
-                if (cells.ContainsKey(cell.index))
-                    cells.Remove(cell.index);
+                cells.Remove(cell.index);
 
-                if (m_ChunkInfo.ContainsKey(cell.index))
-                    m_ChunkInfo.Remove(cell.index);
+                if (cellInfo.loaded)
+                    m_LoadedCells.Remove(cellInfo);
+                else
+                    m_UnloadedCells.Remove(cellInfo);
 
-                if (cell.flatIdxInCellIndices >= 0)
-                    m_CellIndices.MarkCellAsUnloaded(cell.flatIdxInCellIndices);
+                UnloadCell(cellInfo);
```

Comment:
Can we move this in the if (cellInfo.loaded) side of the branch? Would read more logically I think

Reply:
Agreed. The little bit of code above was added later so I did not think of that.